api/scripts/export-ngrok.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import requests
+
+if __name__ == '__main__':
```

Comment:
good call out. I've added some stuff to the readme.

it would be a nicer developer experience to integrate this into config.py but i was wary of coupling it that deeply at this point.